tests/src/test/scala/scalaz/MapTest.scala

```diff
@@ -1003,12 +1003,14 @@ object MapTest extends SpecLite {
   }
 
   type IntMap[A] = Int ==>> A
+   
   checkAll(traverse.laws[IntMap])
   checkAll(bind.laws[IntMap])
   checkAll(align.laws[IntMap])
   checkAll(zip.laws[IntMap])
   checkAll(bifoldable.laws[==>>])
   checkAll(FoldableTests.anyAndAllLazy[IntMap])
+  checkAll(band.laws[ISet[Int]])
```

Comment:
https://github.com/scalaz/scalaz/blob/f23d2f288a00defc73dd796250538f0657146c3c/tests/src/test/scala/scalaz/ISetTest.scala#L21

There is `checkAll(band.laws[ISet[Int]])` in `ISetTest.scala` unnecessary here